docs/pages/index.mdx

```diff
@@ -8,6 +8,12 @@ import "../styles.css";
     internet
   </CustomHomePage.Subhead>
   <CustomHomePage.TileGrid>
+    <CustomHomePage.Tile
+      href="/preview"
+      title="Preview latest docs"
```

Comment:
I love that the new docs are just "the docs" and not "the v3 docs", but in the context of _the present docs_ it feels like they do need to be referred to as v3 docs. I don't get a sense now of why these docs are distinguished and why it matters that they're new.

Reply:
YESSS - I'm so glad you feel this way! 😅 I think this is a perfect example of your suggestion to not call out V3, but add it when there's a tension without it. ON IT.